Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/CapsuleViewportEdit.h

```diff
@@ -20,9 +20,8 @@ namespace AzToolsFramework
     class CapsuleViewportEdit : public BaseShapeViewportEdit
     {
     public:
-        CapsuleViewportEdit(bool allowAsymmetricalEditing = false);
+        CapsuleViewportEdit(bool allowAsymmetricalEditing = false, bool ensureHeightExceedsTwiceRadius = true);
```

Comment:
My preference in this case would be to use 2 enums instead of 2 booleans, it'd be more clear and never make the mistake of messing the order of parameters. I think it's worth it here.

Reply:
I'm going to take it out of the constructor and make a separate function to set it so there's no risk of confusion, but once I do that I don't see any reason not to leave it as a bool.